src/main/java/mate/academy/EventManager.java

```diff
@@ -1,19 +1,38 @@
 package mate.academy;
 
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
 public class EventManager {
-    public void registerListener(EventListener listener) {
+    private final Map<EventListener, Boolean> listeners = new ConcurrentHashMap<>();
```

Comment:
Using a `Boolean` value in the `listeners` map seems unnecessary since you are not utilizing the boolean value at all. If you only need to keep track of the listeners, consider using a `Set<EventListener>` instead.